addons/material-forum-editor-buttons/icons.css

```diff
@@ -0,0 +1,147 @@
+    top: -1px;
+}
+.markItUpHeader ul .markItUpDropMenu {
+    background: transparent;
+}
+.markItUp .markItUpButton12-1 a,
+[src*="//cdn.scratch.mit.edu/scratchr2/static/"][src*="djangobb_forum/img/smilies/smile.png"] {
+    background-image: url("https://easrng.github.io/scratch-emoji-assets/smile.svg");
+}
+.markItUp .markItUpButton12-2 a,
+[src*="//cdn.scratch.mit.edu/scratchr2/static/"][src*="djangobb_forum/img/smilies/neutral.png"] {
+    background-image: url("https://easrng.github.io/scratch-emoji-assets/neutral.svg");
+}
+.markItUp .markItUpButton12-3 a,
+[src*="//cdn.scratch.mit.edu/scratchr2/static/"][src*="djangobb_forum/img/smilies/sad.png"] {
+    background-image: url("https://easrng.github.io/scratch-emoji-assets/sad.svg");
+}
+
+.markItUp .markItUpButton12-4 a,
+[src*="//cdn.scratch.mit.edu/scratchr2/static/"][src*="djangobb_forum/img/smilies/big_smile.png"] {
+    background-image: url("https://easrng.github.io/scratch-emoji-assets/big_smile.svg");
+    }
+
+    .markItUp .markItUpButton12-5 a,
+    [src*="//cdn.scratch.mit.edu/scratchr2/static/"][src*="djangobb_forum/img/smilies/yikes.png"] {
+        background-image: url("https://easrng.github.io/scratch-emoji-assets/yikes.svg");
+    }
+
+    .markItUp .markItUpButton12-6 a,
+    [src*="//cdn.scratch.mit.edu/scratchr2/static/"][src*="djangobb_forum/img/smilies/wink.png"] {
+        background-image: url("https://easrng.github.io/scratch-emoji-assets/wink.svg");
+    }
+
+    .markItUp .markItUpButton12-7 a,
+    [src*="//cdn.scratch.mit.edu/scratchr2/static/"][src*="djangobb_forum/img/smilies/hmm.png"] {
+        background-image: url("https://easrng.github.io/scratch-emoji-assets/hmm.svg");
+    }
+
+    .markItUp .markItUpButton12-8 a,
+    [src*="//cdn.scratch.mit.edu/scratchr2/static/"][src*="djangobb_forum/img/smilies/tongue.png"] {
+        background-image: url("https://easrng.github.io/scratch-emoji-assets/tongue.svg");
+    }
+
+    .markItUp .markItUpButton12-9 a,
+    [src*="//cdn.scratch.mit.edu/scratchr2/static/"][src*="djangobb_forum/img/smilies/lol.png"] {
+        background-image: url("https://easrng.github.io/scratch-emoji-assets/lol.svg");
+    }
+
+    .markItUp .markItUpButton12-10 a,
+    [src*="//cdn.scratch.mit.edu/scratchr2/static/"][src*="djangobb_forum/img/smilies/mad.png"] {
+        background-image: url("https://easrng.github.io/scratch-emoji-assets/mad.svg");
+    }
+
+    .markItUp .markItUpButton12-11 a,
+    [src*="//cdn.scratch.mit.edu/scratchr2/static/"][src*="djangobb_forum/img/smilies/roll.png"] {
+        background-image: url("https://easrng.github.io/scratch-emoji-assets/roll.svg");
+    }
+
+    .markItUp .markItUpButton12-12 a,
+    [src*="//cdn.scratch.mit.edu/scratchr2/static/"][src*="djangobb_forum/img/smilies/cool.png"] {
+        background-image: url("https://easrng.github.io/scratch-emoji-assets/cool.svg");
+    }
+
+    img[src*="//cdn.scratch.mit.edu/scratchr2/static/"][src*="djangobb_forum/img/smilies/"] {
```

Comment:
wouldn't better emojis take care of the emoticons?

Reply:
Keep in mind forum emojis are different

Reply:
Emojis are hard to do with CSS and this is a userstyle. You can use them as svg with a text tag but they will get cut off on some platforms. That's why I use these emojis.